phylogenetic/defaults/clade-i/config.yaml

```
@@ -19,14 +19,14 @@ auspice_name: "mpox_clade-I"
 filter:
   min_date: 1900
   min_length: 100000
-  exclude_where: 'clade!=I'
 
 
-### We don't want to subsample, so specify a config which is essentially a no-op
+### Filter to only Clade I sequences
 subsample:
   everything:
     group_by: ""
     sequences_per_group: ""
+    query: "'clade in [\"I\", \"Ia\", \"Ib\"]'"
```

_(non-blocking)_

Suggestion: use set instead of list because `item in Set` is O(1) while `item in List` is O(n). The different in speed might be negligible for the Clade I build so I wouldn't consider this blocking, but just wanted to flag a potential improvement that can be made elsewhere too.

```suggestion
    query: "'clade in {\"I\", \"Ia\", \"Ib\"}'"
```

Example:

```
In [1]: %timeit "II" in {"I", "Ia", "Ib"}
12.7 ns ± 0.0867 ns per loop (mean ± std. dev. of 7 runs, 100,000,000 loops each)

In [2]: %timeit "II" in ["I", "Ia", "Ib"]
38.1 ns ± 0.52 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
```

Note: creating a set is more costly than creating a list with the same items, but even in the worst case example above where it is created on every loop (unless there are per-run optimizations), the set method is still faster. For `augur filter`, the set should only be created once per metadata chunk so it should be less of a concern.

---

Hmm, when I run the build with this change, I get an error from Pandas

```
ERROR: Internal Pandas error when applying query:
	'Set' nodes are not implemented
Ensure the syntax is valid per <https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#indexing-query>.
```

---

I should've checked with pandas.query first, that's a bummer...